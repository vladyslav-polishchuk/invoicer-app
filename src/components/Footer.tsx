import { Container, Typography } from '@mui/material';
import useScreenSize from '../hooks/useScreenSize';

export default function Footer() {
  const { isMobile } = useScreenSize();
  const copyrightTextVariant = isMobile ? 'caption' : 'body2';
  const padding = isMobile ? 1 : 2;

  return (
    <Container
      sx={{ py: padding, mt: 'auto', displayPrint: 'none', zIndex: 1 }}
      component="footer"
    >
      <Typography
        variant={copyrightTextVariant}
        align="center"
        color="text.secondary"
      >
        Â© Copyright 2022 Vladyslav Polishchuk. All rights reserved
      </Typography>
    </Container>
  );
}
