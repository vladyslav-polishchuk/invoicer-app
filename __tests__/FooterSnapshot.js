import { render } from '@testing-library/react';
import Footer from '../src/components/Footer';

it('Should render footer', () => {
  const { container } = render(<Footer />);

  expect(container).toMatchSnapshot();
});
