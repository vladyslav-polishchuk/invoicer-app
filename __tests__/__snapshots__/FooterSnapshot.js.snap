// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render footer 1`] = `
<div>
  <footer
    class="MuiContainer-root MuiContainer-maxWidthLg css-1rloke0-MuiContainer-root"
  >
    <p
      class="MuiTypography-root MuiTypography-body2 MuiTypography-alignCenter css-1vl95ex-MuiTypography-root"
    >
      Â© Copyright 2022 Vladyslav Polishchuk. All rights reserved
    </p>
  </footer>
</div>
`;
